tasks/manage_zfs.yml

```diff
@@ -90,15 +90,18 @@
       atime: "{{ item.atime|default(omit) }}"
       compression: "{{ item.compression|default(omit) }}"
       dedup: "{{ item.dedup|default(omit) }}"
+      encryption: "{{ item.encryption|default(omit) }}"
+      keyformat: "{{ item.keyformat|default(omit) }}"
+      keylocation: "{{ item.keylocation|default(omit) }}"
       logbias: "{{ item.logbias|default(omit) }}"
       mountpoint: "{{ item.mountpoint|default(omit) }}"
       primarycache: "{{ item.primarycache|default(omit) }}"
       quota: "{{ item.quota|default(omit) }}"
       recordsize: "{{ item.recordsize|default(omit) }}"
       sharenfs: "{{ item.sharenfs|default(omit) }}"
-      sync: "{{ item.sync|default(omit) }}"
       snapdev: "{{ item.snapdev|default('hidden')}}"
       snapdir: "{{ item.snapdir|default('hidden')}}"
+      sync: "{{ item.sync|default(omit) }}"
```

Comment:
I moved some stuff around here and below because these keys were very nearly alphabetized.

Reply:
I definitely appreciate that ;)